packages/lib-classifier/src/store/FieldGuideStore.spec.js

```diff
@@ -35,62 +36,53 @@ const fieldGuideWithoutIcon = FieldGuideFactory.build({
 
 const project = ProjectFactory.build()
 
-describe('Model > FieldGuideStore', function () {
-  function fetchFieldGuide (rootStore) {
-    sinon.stub(rootStore.fieldGuide, 'fetchFieldGuide')
-    rootStore.projects.setResource(project)
-    return rootStore.projects.setActive(project.id)
-      .then(() => {
-        rootStore.fieldGuide.fetchFieldGuide.restore()
-        return rootStore.fieldGuide.fetchFieldGuide()
-      })
-  }
+function setupStores(clientStub) {
+  const store = FieldGuideStore.create()
+  store.projects = ProjectStore.create()
+  store.client = clientStub
+
+  return store
+}
 
-  function setupStores (clientStub) {
-    const store = RootStore.create({
-      classifications: {},
-      dataVisAnnotating: {},
-      drawing: {},
-      feedback: {},
-      subjects: {},
```

Comment:
Should the original code have attached a projects store here? Is that the bug that broke the tests?

Reply:
No, a `ProjectStore` was created on the `RootStore.create` call here. It just wasn't getting set to an empty object as a snapshot like the others because it needed to be set with an actual project for the field guide tests. I still don't understand what changed or broke between the update and the final merging of all the updates.

Reply:
These fail when I run them with `.only` to isolate them so I think they've always been broken. We just haven't noticed because Travis isn't running the tests for any of these PRs.

Reply:
They were working in the update PR for the FieldGuideStore, and I run tests locally before pushing anything up, so I don't know what changed, but between all the updates, scoping, and rebasing, I think it's not really worth the time to exactly pinpoint it.

Reply:
Something to consider for the MST update would be adding the target branches to the Travis config, so that CI runs for those branches as well as master.

Maybe also change the docker image name in the compose files, so that this update builds and uses a different image from master. That might save some time with building the repo again in order to test these branches.